build/patch-cluster-prow.sh

```diff
@@ -2,14 +2,17 @@
 
 set -e
 
-DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
-
 echo "* Check for a running ACM"
-acm_installed_namespace=`oc get subscriptions.operators.coreos.com --all-namespaces | grep advanced-cluster-management | awk '{print $1}'`
-while UNFINISHED="$(oc -n ${acm_installed_namespace} get pods | grep -v -e "Completed" -e "1/1     Running" -e "2/2     Running" -e "3/3     Running" -e "4/4     Running" -e "READY   STATUS" | wc -l)" && [[ "${UNFINISHED}" != "0" ]]; do
-  echo "* Waiting on ${UNFINISHED} pods in namespace ${acm_installed_namespace}..."
-  sleep 5
-done
+acm_installed_namespace=$(oc get subscriptions.operators.coreos.com --all-namespaces | grep advanced-cluster-management | awk '{print $1}')
 
-./build/patch-dev-images.sh
+echo "* Initial Pod state:"
+oc -n "${acm_installed_namespace}" get pods
+
+echo "* Waiting for prerequisite pods:"
+kubectl wait pod -l 'app in (grc,policyreport,multicluster-operators-standalone-subscription)' \
```

Comment:
Is this work?  I thought this one work `$(app in (grc,policyreport,multicluster-operators-standalone-subscription) `

Reply:
The `$(` notation is for a subshell, but this isn't a subshell command (unless I'm misunderstanding your question).

Reply:
It's a string meant to be translated by the `-l` flag for a MatchExpressions.

Reply:
oh I didn't know good to know!

Reply:
For reference:
- https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#equality-based-requirement
- https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#set-based-requirement